src/utils/websockets.js

```diff
@@ -64,7 +64,13 @@ export function connectWS(topic, onMessageCallback) {
     };
     /*eslint-enable no-console */
 
-    this.sockClient.activate({}, subscribe);
```

Comment:
Why do you remove the subscribe callback? Do you want to leave it only onConnect hook ?

Reply:
Because it doesn't take a callback: https://stomp-js.github.io/api-docs/latest/classes/Client.html#activate